### PR TITLE
Feature/57 create contact on user assign

### PR DESCRIPTION
**Pull request by lasong**

### CODE REVIEW

File: src/client/components/SupplierRegistrationEditor/SupplierRegistrationEditor.react.js [hunk 1]
```diff
@@ -89,17 +90,34 @@ class SupplierRegistrationEditor extends Component {
...
         }
 
       if (this.props.onChange) {
         this.props.onChange({ isDirty: false });
       }
 
+      const user = this.props.user;
+      const contact = {
+          contactId: `${this.props.username}_${supplier.supplierId}`,
+          contactType: "SIM",
```
Comment (gr4per) [inline src/client/components/SupplierRegistrationEditor/SupplierRegistrationEditor.react.js]: SIM ? Doesnt make sense. Let's talk about this.

File: src/client/components/App.js [hunk 2]
```diff
@@ -19,6 +19,11 @@ const onboardingSupplier = {
   dunsNo: null,
   registrationNumber: "MI342323"
 };
+const onboardingUser = {
+  firstName: 'John',
+  lastName: 'Doe',
+  email: 'john.doe@ncc.com',
+};
```
Comment (gr4per) [inline src/client/components/App.js]: what is this static info used for ?

File: src/client/components/App.js [hunk 3]
```diff
@@ -19,6 +19,11 @@ const onboardingSupplier = {
...
 const username = 'john.doe@ncc.com';
 const actionUrl = 'http://localhost:8080'
```
Comment (gr4per) [inline src/client/components/App.js]: this hardcoded url is not going to work in most cases?

File: src/client/components/App.js (hunk 2 above)
Comment (lasong) [inline src/client/components/App.js]: It's for the supplier test UI

File: src/client/components/App.js (hunk 3 above)
Comment (lasong) [inline src/client/components/App.js]: same as above: supplier test UI

**Pull request merged**